src/megatron/bridge/training/utils/train_utils.py

```diff
@@ -477,62 +489,90 @@ def training_log(
                 writer.add_scalar(metric, value, iteration)
             if wandb_writer:
                 wandb_writer.log(runtime_report, iteration)
+            if mlflow_logger and runtime_report:
```

Comment:
```suggestion
            if mlflow_logger:
```

For consistency with wandb logic.